src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -33,7 +35,10 @@ internal object AutoMend : Module(
     private val autoThrow by setting("Auto Throw", true)
     private val throwDelay = setting("Throw Delay", 2, 0..5, 1, description = "Number of ticks between throws to allow absorption")
     private val autoSwitch by setting("Auto Switch", true)
-    private val autoDisable by setting("Auto Disable", false, { autoSwitch })
+    private val autoDisableExp by setting("Disable on No Exp Bottle", false, { autoSwitch })
```

Comment:
Just name this as "Auto Disable" and have a setting description explaining that it's only when you run out of exp